src/App.js

```diff
@@ -1,24 +1,74 @@
+import React,{useState,useEffect} from 'react';
 import './App.css';
 
+import Container from './components/container';
+import ItemListContainer from './components/ItemListContainer';
+import TodoFormContainer from './components/TodoFormContainer'
+
+import uuidv4 from './utils/random';
+
 function App() {
+
+  const [todoList,setTodoList] = useState([]);
+
+  useEffect(() => {
+    loadDataFromLocalStorage()
+  },[]);
+
+  useEffect(()=>{
+    const data = todoList;
+    saveToLocalStorage(data);
+  },[todoList])
+
+  // todoList의 isDone state를 변경
+  const handleList = (data) => {
+    const found = todoList.findIndex(element=>element.id===data.id);
+    let myarr = [...todoList];
+    myarr[found]=data;
+    setTodoList(myarr);
+  }
+  
+  // todoList 삭제
+  const handleDelete = (id) => {
+    const newList = todoList.filter(element=>element.id!==id);
+    setTodoList(newList);
+  }
+
+  // form에서 입력받은대로 submit결과를 핸들링
+  const handleSubmit = (input) => {
+    if(input !== undefined && input !== ''){
+      const newTodoList = {
+        id:uuidv4(),
+        text:input,
+        isDone:false
+      }
+      setTodoList(prevList => [...prevList,newTodoList]);
+    }
+  }
```

Comment:
아주 사소한 부분인데요, 네이밍 관련해서 newTodoItem이 더 직관적이지 않을까 생각해봤습니다! 다른 ~List네이밍이 붙은 객체들은 전부 배열인 것 같아서요!
```suggestion
      const newTodoItem = {
        id:uuidv4(),
        text:input,
        isDone:false
      }
      setTodoList(prevList => [...prevList,newTodoItem]);
    }
  }
```

Reply:
코드 하나하나 꼼꼼하게 읽어주셔서 감사합니다!